src/servers/ZoneServer2016/classes/vehicle.ts

```diff
@@ -48,6 +48,9 @@ export class Vehicle2016 extends Vehicle {
   ) {
     super(worldId, characterId, transientId, modelId, position, rotation);
     this.npcData.vehicleId = getVehicleId(modelId);
+    this.isInvulnerable = 
+    this.npcData.vehicleId==1337 ||
+    this.npcData.vehicleId==13?true:false
```

Comment:
i don't get the point of this

Reply:
Think its about observer, its using vehicle id 1337 which blows up when you get close to a wall without it

Reply:
it makes parachutes and spectator vehicle invincible, i'm fixing the logic though because the ternary statement there is useless